hw5-1/stack.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <stdlib.h>
+
+
+typedef struct StackElement {
+    int value;
+    struct StackElement *prev;
+} StackElement;
+
+typedef struct {
+    StackElement *lastElement;

```

Comment:
```suggestion
    StackElement *firstElement;
```
Как правило, вершину стека считают первым элементом, а не последним.